test/src/mbgl/test/test.cpp

```diff
@@ -7,6 +7,10 @@
 namespace mbgl {
 
 int runTests(int argc, char *argv[]) {
+#if TEST_HAS_SERVER && defined(__QT__)
```

Comment:
Maybe it's better to run the Node.js server.

I think we want to get rid of the C++ server because of it's limitations.

Reply:
What limitations exactly? I can run unit tests just fine and I do not want to depend on Node.js.

Reply:
Limited concurrent connections apparantly https://github.com/maplibre/maplibre-gl-native/pull/674#discussion_r1089674583

Reply:
I'd keep this one for Qt if I may until we clarify why we even need that many connections for testing? I guess to run tests in parallel (although it's not obvious to me if we use that)?